scripts/update_VGA.sh

```diff
@@ -51,8 +51,24 @@ else
     if [ -r /media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run ]; then
       VGArun=/media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run
     else
-        echo "CD still not found. Aborting!" >> /dev/stderr
-        exit
+        echo "CD still not found. Trying /dev/sr0 !" >> /dev/stderr
+
+        if [ ! -e /media/cdrom ]; then
+          mkdir -p /media/cdrom
+        fi
+
+        mount /dev/sr0 /media/cdrom
+
+        ERRORCODE=$?
+        if [ $? -ne 1 ]; then
```

Comment:
`ERRORCODE` not used, but in any case testing if it's 1 seems dangerous. who knows what `mount` will return. I think much better to do
```sh
if mount -t iso9660 /dev/sr0 /media/cdrom/
then
 ...
else
...
fi
```